README.md

```diff
@@ -74,7 +74,7 @@ import (
 
 // Option 1: Pass in the values yourself
 opts := gophercloud.AuthOptions{
-  IdentityEndpoint: "https://my-openstack.com:5000/v2.0",
+  IdentityEndpoint: "https://openstack.example.com:5000/v2.0",
```

Comment:
Bah. I explained in the individual commit, but the PR doesn't really show that ...

"RFC 2606 defines example.com as a reserved domain to be used in tests
and examples. It is guaranteed to not be actually resolvable. Update
the uses of my-openstack.com to use example.com instead."

Essentially, there's a domain that's built in to the RFCs for this.